### ISSUE TITLE
Can't resolve 'fs'

### ISSUE THREAD

**Issue by darnfish** [neet/masto.js#469]
I can't use this library in a simple Next.js project.

<img width="383" alt="Screenshot 2021-08-29 at 8 15 05 am" src="https://user-images.githubusercontent.com/8293444/131242040-b7d55b82-6c52-4348-bad5-ba633975a796.png">

### pages/index.tsx
```js
import { useEffect, useState } from 'react'

import { login } from 'masto'

export default function HomePage() {
	const [user, setUser] = useState(null)

	async function loadUser() {
	  const masto = await login({
	    url: 'https://mastodon.social',
	    accessToken: 'bruh',
	  })

	  const selfUser = await masto.accounts.lookup({ acct: 'darnfish' })

	  console.log({ selfUser })
	}

	useEffect(() => {
		loadUser()

		return () => {}
	}, [])

	return <p>test</p>
}
```

### .babelrc
```
{
  "presets": [
    "@babel/preset-env",
    "@babel/preset-react"
  ]
}
```

**Comment by neet** [comment]
Hi @darnfish, thanks for reporting an issue. 

I confirmed that this error occurs at the master by using module bundlers that support ESM (like Next.js). At the first, I thought this is happening because I didn't put `node:` prefix before `fs` like `node:fs` but adding it did not help to fix the error. Repro: https://github.com/neetbox/next-masto-469-repro

This seemed to be occurring because one of the Masto dependency `isomorphic-form-data` directly depends on Node.js' `fs` module regardless of whether you're making a browser app or not, so you cannot resolve it on the browser environment. Now I think changing `isomorphic-form-data` to peer-dependency to make Node.js users install an isomorphic one with fs module on demand, may resolve this issue.

I'll look into it later but since I'm not yet familiar with ESM, it would take a while. Of course, you can also feel free to open PR for it ;)

**Comment by neet** [comment]
Update: Since isomorphic-form-data provides `browser` field and `main` field on `package.json`, module bundlers like webpack should automatically switch Node.js module form-data and browser native FormData. However, Next.js seems not the case.

Removing Node.js form-data and forces the Node.js users to `import 'form-data'` on the top of the code manually would be unintuitive so I'd like to avoid the verbosity as far as I can.

So for the Next.js users, I released @darnfish's patch under a 5.0.0-rc1 which is under a `next` tag. You can install the fixed version with:

```
npm install masto@next
```

Note that this is a `next` version so it would not be stable and breaking changes will be applied without any announcements. I'll go on researching the nicest way to resolve this.

**Comment by darnfish** [comment]
Working on my end, thank you so much!
